modules/osdk-cli-ref-scorecard.adoc

```diff
@@ -31,6 +31,9 @@ The `operator-sdk scorecard` command runs the scorecard tool to validate an Oper
 |`-o`, `--output` (string)
 |Output format for results. Available values are `text`, which is the default, and `json`.
 
+|`--pod-security <security_context>`
+|Option to run scorecard with the specified security context. Allowed values include `restricted` and `legacy`. The default value is `legacy`. ^[1]^
```

Comment:
```suggestion
|The option to run scorecard with the specified security context. Allowed values include `restricted` and `legacy`. The default value is `legacy`. ^[1]^
```
I like the definite article with words like "option." Totally preference, up to you.